15-eco.Rmd

```diff
@@ -519,7 +519,7 @@ search_space = paradox::ps(
 Having defined the search space, we are all set for specifying our tuning via the `AutoTuner()` function.
 Since we deal with geographic data, we will again make use of spatial cross-validation to tune the hyperparameters\index{hyperparameter} (see Sections \@ref(intro-cv) and \@ref(spatial-cv-with-mlr)).
 Specifically, we will use a five-fold spatial partitioning with only one repetition (`rsmp()`). 
-In each of these spatial partitions, we run 50 models (`trm()`) while using randomly selected hyperparameter configurations (`tnr`) within predefined limits (`seach_space`) to find the optimal hyperparameter\index{hyperparameter} combination.
+In each of these spatial partitions, we run 50 models (`trm()`) while using randomly selected hyperparameter configurations (`tnr()`) within predefined limits (`seach_space`) to find the optimal hyperparameter\index{hyperparameter} combination.
```

Comment:
Definite improvement, follow-on question, worth explaining in more detail what these functions are, I'm new to them and am not sure from this good but terse description.

Reply:
Good point, will reference the spatial-cv chapter as I have explained there in a little more detail how to construct an `AutoTuner()`.